.changeset/three-pumas-help.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fix an issue where config for `/index.js` could not be loaded
```

Comment:
Something like

```suggestion
Fixed: ensure that `resolveSilent` util resolves `.` as `index.js`
```

would be more accurate.

Reply:
Thanks for the suggestion. But I think `resolveSilent` is internal, so I'd like to avoid it if possible.